Shared/AppConstants.swift

```diff
@@ -114,6 +114,18 @@ public struct AppConstants {
         return false
         #endif
     }()
+    
+    public static let IS_RECENTLY_SAVED_SECTION_ENABLED: Bool = {
+        #if MOZ_CHANNEL_RELEASE
+        return false
+        #elseif MOZ_CHANNEL_BETA
+        return false
```

Comment:
```suggestion
        return true
```

Make the beta to be true to so we can run diary study if necessary